lib/koans/07_pattern_matching.ex

```diff
@@ -5,13 +5,18 @@ defmodule PatternMatching do
     assert match?(1, ___)
   end
 
-  koan "A pattern can change" do
+  koan "A value can be bound to a variable" do
+    a = 1
+    assert a == ___
+  end
+
+  koan "A variable can be rebound" do
     a = 1
     a = 2
     assert a == ___
   end
 
-  koan "A pattern can also be strict" do
+  koan "A variable can be pinned to be prevent it from being rebound" do
```

Comment:
It seemed that this description unnecessarily danced around the terminology "pin".